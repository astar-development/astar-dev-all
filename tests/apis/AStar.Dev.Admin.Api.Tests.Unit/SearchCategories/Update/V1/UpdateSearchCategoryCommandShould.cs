using AStar.Dev.Api.Usage.Sdk.Metrics;
using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Admin.Api.SearchCategories.Update.V1;

// [TestSubject(typeof(UpdateSearchCategoryCommand))]
public sealed class UpdateSearchCategoryCommandShould
{
    [Fact]
    public void ContainTheExpectedValues()
    {
        var sut = new UpdateSearchCategoryCommand();

        sut.ToJson().ShouldMatchApproved();
    }

    [Fact]
    public void ImplementIEndpointNameInterface()
    {
        var sut = new UpdateSearchCategoryCommand();

        sut.ShouldBeAssignableTo<IEndpointName>();
    }
}
