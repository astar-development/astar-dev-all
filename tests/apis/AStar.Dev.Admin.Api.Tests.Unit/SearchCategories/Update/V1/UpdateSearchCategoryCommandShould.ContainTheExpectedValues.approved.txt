{
  "order": 0,
  "categoryName": "",
  "lastKnownImageCount": 0,
  "lastPageVisited": 0,
  "totalPages": 0
}